milestones/serializers.py

```diff
@@ -66,3 +69,23 @@ def deserialize_milestone(milestone_dict):
         namespace=milestone_dict.get('namespace', ''),
         description=milestone_dict.get('description', '')
     )
+
+
+def serialize_requirements(requirements):
+    """
+    Convert JSON serializable object to string
+    """
+    if requirements is not None:
+        requirements = json.dumps(requirements)
+    return requirements
+
+
+def deserialize_requirements(requirements):
```

Comment:
So is the idea here that the 'requirements' value is going to be a JSON dict or some sort of list?  I guess I thought it was going to be a simple string field/value?